### PR TITLE
HOC & DT: Tweak class title bar hover state

### PR DESCRIPTION
**Pull request by adamkecskes**
New hover area defined for the border. Pls check difference on screen recording below:
BEFORE / AFTER
all-classes-hover

### CODE REVIEW

File: ozaria/site/components/teacher-dashboard/BaseMyClasses/components/ClassSummaryRow.vue [hunk 1]
```diff
@@ -217,12 +219,33 @@
     align-items: center;
   }
 
+  .class-hover-area {
+    width: 100%;
+    &:hover {
```
Comment (mrfinch) [inline ozaria/site/components/teacher-dashboard/BaseMyClasses/components/ClassSummaryRow.vue:224]: can we remove the previous hover code too?

**Review by mrfinch — COMMENTED** [review]
I think we need to cover Share button too on hover. Screenshot in asana is old when Share wasn't launched that's why it doesn't reflect Share button. 

Including hover on Share will complete goal of this asana more appropriately: ```When user hovers their mouse over the class title bar, the blue outline hover state should appear around the entire class title bar```

File: ozaria/site/components/teacher-dashboard/BaseMyClasses/components/ClassSummaryRow.vue (hunk 1 above)
Comment (adamkecskes) [reply ozaria/site/components/teacher-dashboard/BaseMyClasses/components/ClassSummaryRow.vue:224]: @mrfinch It's not clear for me what code to remove.
The `border` was removed from line 225, but we still need the `cursor: pointer.`

**Comment by adamkecskes** [comment]
@mrfinch share button is now inside: 
all-classes-hover-share

**Review by mrfinch — APPROVED** [review]

**Pull request merged**